### ISSUE TITLE
Revise Template metadata to only have one version number (Cicero+Ergo)

### ISSUE THREAD

**Issue by jeromesimeon** [accordproject/cicero#466]
The plan moving forward is to publish Ergo and Cicero in lock-step with the same version number. This makes the current metadata redundant. The suggestion is to consolidate into a single `version` field in the `accordproject` part of the `package.json`.

So to move from:
```
  "accordproject": {
    "template": "clause",
    "ergo": "0.9.0",
    "cicero": "^0.13.0",
    "runtime": "ergo"
  }
```
to:
```
  "accordproject": {
    "template": "clause",
    "version": "0.20.0",
    "runtime": "ergo"
  }
```

Note: the `runtime` component is still important to distinguish between source archive (containing `ergo`) and compiled archiver (containing `javascript` or possibly other executable code).

**Comment by dselman** [comment]
This reads to me like “accord project version” which is not what we want imo. How about ciceroVersion as the attribute?

**Comment by jeromesimeon** [comment]
> This reads to me like “accord project version” which is not what we want imo. How about ciceroVersion as the attribute?


How about `cicero` since that's what we currently use. This definitely makes it easier to implement and easier for user migration!

(Thanks @dselman )

**Comment by jeromesimeon** [comment]
Unless I hear otherwise, I will implement:
```
  "accordproject": {
    "template": "clause",
    "cicero": "0.20.0",
    "runtime": "ergo"
  }
```
I suspect this means no code change (I am unclear we use the Ergo version number), but I will remove the `ergo` version from tests and templates, and add a mention in the migration guide. IMO this will reduce confusion.

**Comment by jeromesimeon** [comment]
Small update on this:
- Removed `getErgoVersion()` API call on `Metadata` class
- Removed `ergo` version field in all `package.json`
- Now determines wether source or compiled archive through `getRuntime()` call

**Comment by jeromesimeon** [comment]
Fixed in #468
